layouts/New/layouts.module.css

```diff
@@ -58,32 +58,25 @@
   }
 }
 
-.homeLayout {
+.centeredLayout {
```

Comment:
I think you misunderstood here what I was referring.

The `centeredLayout` should be a base for the "homeLayout",

meaning that the centered layout is the layout that has all the centering and stuff and padding and etc

And the home layout only styles the sections within the `main` element.

Meaning homeLayout actually extends the centeredLayout:

```tsx
<CenteredLayout>
  <main className={styles.homeLayout}>
    {children}
  </main>
</CenteredLayout>
```

Reply:
Meaning, try to remove as much as duplicated styles as possible. And in other words the "homeLayout" styles only handle the styling of `main` itself and the sections within

And `.centeredLayout` handles the container and etc